trace/summary.go

```diff
@@ -0,0 +1,85 @@
+package trace
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/fatih/color"
+	"github.com/temporalio/cli/common"
+	"github.com/urfave/cli/v2"
+	sdkclient "go.temporal.io/sdk/client"
+)
+
+var (
+	bold = color.New(color.Bold)
+)
+
+// Table is a list of rows to be printed with the same width on the first column
+type Table struct {
+	rows []Row
+	w    io.Writer
+}
+
+type Row struct {
+	key   string
+	value string
+}
+
+func NewTable(w io.Writer) *Table {
+	t := new(Table)
+	t.w = w
+	return t
+}
+
+// padRight pads a text to the right with spaces to fill the padding width.
+func padRight(text string, padding int) string {
+	return fmt.Sprintf("%-*s", padding, text)
+}
+
+func (t *Table) Print() error {
+	// Figure out which is the widest key
+	maxWidth := 0
+	for _, row := range t.rows {
+		if l := len(row.key); l > maxWidth {
+			maxWidth = l
+		}
+	}
+	for _, row := range t.rows {
+		_, err := fmt.Fprintf(t.w, "  %s : %s\n", bold.Sprintf(padRight(row.key, maxWidth)), row.value)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// AppendRows appends a list of rows to the table
+func (t *Table) AppendRows(rows ...Row) {
+	t.rows = append(t.rows, rows...)
+}
+
+func PrintWorkflowSummary(c *cli.Context, sdkClient sdkclient.Client, wfId, runId string) error {
+	tcCtx, cancel := common.NewIndefiniteContext(c)
+	defer cancel()
+
+	res, err := sdkClient.DescribeWorkflowExecution(tcCtx, wfId, runId)
+	if err != nil {
+		return err
+	}
+
+	info := res.GetWorkflowExecutionInfo()
+
+	_, _ = title.Println("Execution summary:")
+	tb := NewTable(os.Stdout)
+	tb.AppendRows(
```

Comment:
this seems similar to an existing API, https://github.com/temporalio/cli/blob/f8c84edae78379ad3716ed51c84120c923db9388/cluster/cluster_commands.go#L61

Have you also considered using it?

Reply:
yea it was not documented. Designed this way to accept user's input in a simple form
Let me look into documenting this better